packages/react-ui-components/tsconfig.json

```diff
@@ -1,27 +1,21 @@
 {
 	"compilerOptions": {
-		"strict": true,
-		"allowUnreachableCode": false,
-		"allowSyntheticDefaultImports": true,
 		"declaration": true,
+		"noImplicitAny": false,
```

Comment:
are we sure that we want this config define as false?

Reply:
No, this was a leftover as well, will revert this